alphagov/tsuru-performance-tests
# [#99680570] Feature/populate elastic search

# What

Added the search API app to the list of deployed apps and a task to import Elasticsearch data.

The deployment script works the same way as explained in the README file.

All frontend apps have been reconfigured to point to the correct Elasticsearch endpoint.
# How To Test

On a clean environment in working state (check that Elasticsearch service and broker are running after resuming a previously stopped environment):

```
bundle install
bundle exec rake \
    admin_pass=mysecretpass \
    environment=test host=tsuru.paas.alphagov.co.uk \
    num_users=10 num_teams=2 \
    dm_api_pg_dump_url="http://somehost.com/full.dump" \
    dm_api_pg_dump_auth_header="Auth: ..." \
    deploy  
```
# Who To Review

Somebody other than @RichardKnop
